marvin-ai/marvin-engine-executor
# Tk execution status

To test use the iris engine:

1.  Send a batch execution request
`curl -H "Content-Type: application/json" -X POST http://localhost:8000/acquisitor -d '{}'`
3. Get the given protocol and send a request to check status
 `curl -H "Content-Type: application/json" -X GET http://localhost:8000/acquisitor/status?protocol=???? -d '{}' `
5. Check the response (status must be **working**)
7. Change the acquisition code to sleep for 60 seconds
8. Restart everything
9. Send a batch execution request get the protocol and check status again (before sleep finished)
10.Check the response (status must be **failed**)
